src/adhocracy_mercator/adhocracy_mercator/scripts/export_users.py

```diff
@@ -43,27 +88,39 @@ def export_users():
 
     users = query.execute()
 
+    # FIXME: set 100 instead of 0 on prod
```

Comment:
You don't need the catalog to find all users. Instead get the users service:

    from substanced.util import find_service
    users = find_service(context, 'principals', 'users')


and iterate all children.